lib/decompress.ps1

```diff
@@ -149,20 +167,23 @@ function Expand-InnoArchive {
         [Parameter(Position = 1)]
         [String]
         $DestinationPath = (Split-Path $Path),
+        [String]
+        $ExtractDir,
         [Parameter(ValueFromRemainingArguments = $true)]
         [String]
         $Switches,
         [Switch]
         $Removal
     )
+    $DestinationPath = $DestinationPath.TrimEnd("\")
     $LogPath = "$(Split-Path $Path)\innounp.log"
-    $ArgList = @('-x', "-d`"$DestinationPath`"", "-c`{app`}", "`"$Path`"", '-y')
+    $ArgList = @('-x', "-d`"$DestinationPath`"", "-c`{app`}\$ExtractDir", "`"$Path`"", '-y')
```

Comment:
db2 list database directory show detail

Reply:
It will work, but `$DestinationPath` in some functions will not work... So I add it to every functions.

Reply:
A bit late but:
`$ExtractDir` should use `{app}` as a default value instead of hardcoding it to `{app}\$ExtractDir`.
For example, fixing `freedownloadmanager` requires files from `{app}` and `{code_CefInstallDir}` to be extracted and merged.
This hardcoded value would require running:
```pwsh
Invoke-ExternalCommand (Get-HelperPath -Helper Innounp) @('-x', "-d`"$DestinationPath`"", -c"`{app`}", "`"$Path`"", '-y')
Invoke-ExternalCommand (Get-HelperPath -Helper Innounp) @('-x', "-d`"$DestinationPath`"", -c"`{code_CefInstallDir`}", "`"$Path`"", '-y')
```
instead of simply:
```pwsh
Expand-InnoArchive -Path \"$dir\\$fname\" -ExtractDir "{app}"
Expand-InnoArchive -Path \"$dir\\$fname\" -ExtractDir "{code_CefInstallDir}"
```

See:
https://github.com/lukesampson/scoop-extras/pull/2137
https://github.com/lukesampson/scoop-extras/issues/2150

Reply:
Same for GOG Galaxy

https://github.com/Ash258/scoop-Ash258/blob/master/bucket/GOGGalaxy.json

Reply:
Same for https://github.com/Ash258/scoop-Ash258/blob/master/bucket/GOGGalaxy.json

Could be simplified to just two calls of `Expand-InnoArchive`